elpy.el

```diff
@@ -3309,13 +3313,165 @@ Returns a possible multi-line docstring."
             success error))
 
 (defun elpy-rpc-get-usages (&optional success error)
+  "Return the symbol under point usages as a list"
   (elpy-rpc "get_usages"
             (list buffer-file-name
                   (elpy-rpc--buffer-contents)
                   (- (point)
                      (point-min)))
             success error))
 
+(defun elpy-rpc-get-names (&optional success error)
+  "Return all names (possible candidates for jumping to definition ot others)."
+  (elpy-rpc "get_names"
+            (list buffer-file-name
+                  (elpy-rpc--buffer-contents)
+                  (- (point)
+                     (point-min)))
+            success error))
+
+;;;;;;;;;;;;;;;;
+;;; Xref backend
+(when (featurep 'xref)
```

Comment:
You do not need to conditionally define these. As long as they are not used, they can happily be defined, I think?